proxy.conf.json

```diff
@@ -1,12 +1,12 @@
 {
     "/v1beta": {
```

Comment:
Generic question: Why the IPs are hardcoded? I mean what IPs are these?
Also, as in entrypoint.json, it has localhost API as DELFIN_URL, so to separate delfin n dashboard on different nodes is not possible

Reply:
@kumarashit This is the file used for local development. Updated it in this PR as anyone wishing to work on delfin ui will need these changes.